salt/metalk8s/addons/logging/loki/deployed/init.sls

```diff
@@ -86,6 +87,20 @@ Ensure Loki datasource does not exist:
     - require:
       - salt: Ensure Loki objects does not exist
 
+Ensure Loki dashboard does not exist:
+  metalk8s_kubernetes.object_absent:
+    - name: loki-dashboard
+    - namespace: metalk8s-monitoring
+    - apiVersion: v1
+    - kind: ConfigMap
+
+Ensure logs dashboard does not exist:
+  metalk8s_kubernetes.object_absent:
+    - name: logs-dashboard
+    - namespace: metalk8s-monitoring
+    - apiVersion: v1
+    - kind: ConfigMap
```

Comment:
Use the same logic has for the charts, you just have to add the dashboard state in `mods` list above